pkg/apis/pingcap/v1alpha1/tidbclusterautoscaler_types.go

```diff
@@ -112,6 +112,24 @@ type BasicAutoScalerSpec struct {
 	// If not set, the default metric will be set to 80% average CPU utilization.
 	// +optional
 	Metrics []v2beta2.MetricSpec `json:"metrics,omitempty"`
+
+	// MetricsTimeDuration describe the Time duration to be queried in the Prometheus
```

Comment:
What's the format of this parameter? Any default value?

Reply:
The format should be time.Duration validated. The check would be realized in calculation request.